npm/private/utils.bzl

```diff
@@ -670,7 +104,8 @@ def _npm_registry_download_url(package, version, registries, default_registry):
         registry.removesuffix("/"),
         package,
         package_name_no_scope,
-        _strip_v5_peer_dep_or_patched_version(version),
+        # Strip the rules_js peer/patch metadata off the version. See pnpm.bzl
+        version[:version.find("_")] if version.find("_") != -1 else version,
```

Comment:
This is the one actual change in this file, inlining that `_strip_v5_peer_dep_or_patched_version` method to here. It's inlined so no functional change, but this method is called with data in the rules_js format and not in the pnpm lockfile format so I think this is better anyway 🤷